### ISSUE TITLE
Question: Any solution to add to context freshly created templates?

### ISSUE THREAD

**Issue by mrlss** [radiocity/twig-html-loader#29]
Hello. Is there any chance to add newly created template to context, so plugin can compile it without reloading build?

**Comment by radiocity** [comment]
AFAIK, no. [twig-loader](https://github.com/zimmo-be/twig-loader#dynamic-templates-and-registering-at-runtime) should help.

**Comment by xavierfoucrier** [comment]
I agree with that, unfortunately, it's not possible to load fresh view/layout/template into the current context...
It would be good to "watch", I guess, only Twig "namespaces", and prevent the user to manually stop/reload the current build.
